pkg/controller/pxc/controller.go

```diff
@@ -1109,3 +1099,40 @@ func createOrUpdateConfigmap(cl client.Client, configMap *corev1.ConfigMap) erro
 
 	return nil
 }
+
+func updateServiceIfNeeded(cl client.Client, old, new *corev1.Service) error {
+	if !isServicesSpecEqual(&old.Spec, &new.Spec) || !reflect.DeepEqual(old.Annotations, new.Annotations) ||
+		!reflect.DeepEqual(old.Labels, new.Labels) || !reflect.DeepEqual(new.Spec.Selector, old.Spec.Selector) {
+		log.Info("UPDATING HAPROXY", "name", new.Name)
```

Comment:
This log message looks aggressive :smile:

Reply:
removed there)
068a20ad